src/kbmod/search/kernels.cu

```diff
@@ -147,8 +147,8 @@ __global__ void searchFilterImages(int num_images, int width, int height, void *
         for (int i = 0; i < num_images; ++i) {
             // Predict the trajectory's position.
             float curr_time = image_data.image_times[i];
-            int current_x = x + int(curr_trj.vx * curr_time + 0.5);
-            int current_y = y + int(curr_trj.vy * curr_time + 0.5);
+            int current_x = x + (int)(floor(curr_trj.vx * curr_time));
+            int current_y = y + (int)(floor(curr_trj.vy * curr_time));
```

Comment:
Minor question: From the earlier discussion, it seems the "+ 0.5" is used so that the int casting will round the result rather than truncate/floor it.

We should definitely be consistent about whichever we do, but out of curiosity is there a reason to prefer flooring over rounding?

Reply:
After writing that, I'm realizing that I shouldn't have used truncate/floor interchangeably since they are only equivalent operations for positive values.